internal/engine/eval/jq/jq.go

```diff
@@ -42,11 +42,11 @@ func NewJQEvaluator(assertions []*pb.RuleType_Definition_Eval_JQComparison) (*Ev
 
 	for idx := range assertions {
 		a := assertions[idx]
-		if a.Profile == nil {
-			return nil, fmt.Errorf("missing profile accessor")
+		if a.Profile == nil && a.Constant == nil {
+			return nil, fmt.Errorf("missing profile or constant accessor")
 		}
```

Comment:
Do you want `Profile` and `Constant` to be XOR (at least one set, but not both)?

Reply:
Good point, I've updated it